Makefile

```diff
@@ -20,11 +20,11 @@ test-docker:
 	@# run tests inside a docker container
 	docker run --rm -v $(PWD):/host -w /host --network="host" vapor/swift:5.2-bionic make test
 
-test-e2e: reset reconcile ingest
+test-e2e: db-reset reconcile ingest analyze
 	@# run import sequence test
```

Comment:
Is there a reason analyze wasn't part of this? My guess is just an oversight as it was added?

Reply:
Ah yes, I used this target a lot early on but by the time I started testing analyse I really only needed to run that in isolation - good to have it in!